examples/llm/megatron_gpt_pretraining.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+
+from pytorch_lightning.loggers import TensorBoardLogger
+
+from nemo import lightning as nl
+from nemo.collections import llm
+from nemo.collections.llm.api import train
+from nemo.collections.llm.gpt.data import PreTrainingDataModule
+from nemo.collections.nlp.modules.common.tokenizer_utils import get_nmt_tokenizer
+from nemo.lightning import NeMoLogger
+from nemo.lightning.pytorch.callbacks import ModelCheckpoint
+
```

Comment:
Can we add a comment here explaining the purpose of this script?

Reply:
Done